exonum-java-binding-time-service/pom.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.exonum.binding</groupId>
+        <artifactId>exonum-java-binding-parent</artifactId>
+        <version>0.4-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>exonum-java-binding-time-service</artifactId>
+    <packaging>jar</packaging>
+
+    <name>Exonum Time Service</name>
+    <url>https://exonum.com/doc/version/latest/advanced/time/</url>
+
+    <properties>
```

Comment:
This POM does not seem to be formatted according to our conventions (use 2 spaces, and apply tidy-pom).